genesis/utils/urdf.py

```diff
@@ -358,7 +358,42 @@ def parse_urdf(morph, surface):
     for j_info in j_infos:
         j_info["pos"] *= morph.scale
 
-    return l_infos, j_infos
+    equalities = parse_equality(robot, morph, j_infos)
+
+    return l_infos, j_infos, equalities
+
+
+def parse_equality(robot, morph, j_infos):
+    equalities = []
+
+    for joint in robot.joints:
+        if joint.mimic:
+            print(
+                f"Joint '{joint.name}' mimics '{joint.mimic.joint}' with multiplier {joint.mimic.multiplier} and offset {joint.mimic.offset}"
```

Comment:
Try keeping all lines shorter than 120 characters. This is unreadable in review.